src/odemis/acq/fastem.py

```diff
@@ -84,6 +84,11 @@
 }
 
 
+class ROASkipped(Exception):
+    def __init__(self, message):
+        super().__init__(message)
```

Comment:
If you just call `super()`, it should be equivalent to to just write:
```suggestion
class ROASkipped(Exception):
    pass
```